### PR TITLE
Update superface one-sdk and cli dependencies

### PR DESCRIPTION
**Pull request by janhalama**
## Description

This PR updates @superfaceai/cli to version 0.0.25 and @superfaceai/one-sdk to version 0.0.36

## Motivation and Context

@superfaceai/one-sdk 0.0.36 introduced breaking changes that had to be fixed in this PR. `Profile` definition has been changed and `SuperJsonDocument` has been moved to @superfaceai/ast.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have read the **CONTRIBUTING** document.
- [x] I haven't repeated the code. (DRY)
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: src/util.ts [hunk 1]
```diff
@@ -1,3 +1,4 @@
+import { ProfileId } from '@superfaceai/cli/dist/common/profile';
```
Comment (Jakub-Vacek) [inline src/util.ts:1]: ProfileId (and MapId) helper will be moved to AST to enable using them in more of our projects. I will try to do release today :)

File: src/util.test.ts [hunk 2]
```diff
@@ -84,15 +85,21 @@ describe('util', () => {
       expect(result).toEqual([
         {
           profile: {
-            scope: 'scope',
-            name: 'name',
+            id: {
+              id: 'scope/name',
+              scope: 'scope',
+              name: 'name',
+            },
```
Comment (freaz) [inline src/util.test.ts:92]: You shouldn't change result shape.
This change breaks anything using this function. eg. [validate](https://github.com/superfaceai/station/blob/main/src/validate.test.ts#L16)

**Review by freaz — CHANGES_REQUESTED** [review]
Breaking change in `allProfileProviderCombinations`

File: src/util.test.ts (hunk 2 above)
Comment (janhalama) [reply src/util.test.ts:92]: This is just the format of profile `check` function from `@superfaceai/cli` accepts.

Comment (janhalama) [reply src/util.test.ts:92]: But right the profile format from `@superfaceai/cli` is complex and we can map to it before `check` function is called.

Comment (freaz) [reply src/util.test.ts:92]: No no, you have it right. I overlooked https://github.com/superfaceai/station/pull/82/files#diff-017358cdc577c3a0f1d5f0d4a96fb167e4a4d1705958540e6a2bd6770873ed6dR17

**Review by freaz — APPROVED** [review]
Looks good.  👍 Sorry for my mistake.

File: src/util.ts (hunk 1 above)
Comment (janhalama) [reply src/util.ts:1]: @Jakub-Vacek may I merge this now and import ProfileId helper from AST later?

Comment (Jakub-Vacek) [reply src/util.ts:1]: @janhalama definitely :)

**Review by Jakub-Vacek — DISMISSED** [review]

**Comment by janhalama** [comment]
@Jakub-Vacek I need (hopefully) last approve. I rebased all commits and signed them with gpg.

**Review by Jakub-Vacek — APPROVED** [review]

**Pull request merged**